### PR TITLE
Upstream CI scripts

### PR DESCRIPTION
**Pull request by AshwinHIBM**
Scripts to run tests on upstream moby repository
1. Check kernel configuration
2. Build dev image
3. Unit tests
4. Integration tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ The corresponding prow jobs are:
 2. postsubmit-build-test-containerd
 3. postsubmit-test-docker-staging
 4. postsubmit-test-docker-release
+5. periodic-config-docker
```
Comment (clnperez) [inline README.md:14]: Can you make a new paragraph here and introduce these new jobs? The first are for building and testing docker-ce packages. And the new ones are for running the upstream CI tests. I'd just like for it to be clear that they're not related to the first 4.

File: upstream-master-ci/integration-tests.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                                  -e TESTDEBUG \
+                                  -e TEST_INTEGRATION_USE_SNAPSHOTTER \
+                                  -e TEST_SKIP_INTEGRATION_CLI \
+                                  -e TIMEOUT \
+                                  -e VALIDATE_REPO=${GIT_URL} \
+                                  dockerbuildimage \
+                                  hack/make.sh \
+                                    dynbinary \
+                                    test-integration
+exit 0
```
Comment (clnperez) [inline upstream-master-ci/integration-tests.sh:29]: can you point me to where this docker run command came from?

**Review by clnperez — CHANGES_REQUESTED** [review]
i'm wondering if there is less we can embed as far as scripts, but overall this looks good. if you could just answer the one question i asked i'll let you know if i think this is okay to merge (after the little readme update)

File: upstream-master-ci/integration-tests.sh (hunk 2 above)
Comment (AshwinHIBM) [reply upstream-master-ci/integration-tests.sh:29]: https://github.com/moby/moby/commit/59aa3dce8a3abc9d6eb29df7c76b4484e8758234#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8L312

File: README.md (hunk 1 above)
Comment (AshwinHIBM) [reply README.md]: Modified: https://github.com/ppc64le-cloud/docker-ce-build/pull/176/commits/b09644d1e41d75057ea11b4c5afa574bbb03d351

**Pull request merged**